transports/websys/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "libp2p-websys-transport"
+edition = "2021"
+rust-version = "1.60.0"
+description = "libp2p transports for WASM browser runtime."
+version = "0.2.0"
+authors = ["Vince Vasta <vince.vasta@gmail.com>"]
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[dependencies]
+futures = "0.3.26"
+js-sys = "0.3.61"
+libp2p-core = { version = "0.39", path = "../../core" }
+parking_lot = "0.12.1"
+send_wrapper = "0.6.0"
+thiserror = "1.0.38"
+wasm-bindgen = "0.2.84"
+web-sys = { version = "0.3.61", default-features = false }
+
+[features]
+websocket = ["web-sys/BinaryType", "web-sys/CloseEvent", "web-sys/MessageEvent", "web-sys/WebSocket"]
```

Comment:
As written above, I think we can remove this feature, can't we? Why would one use this transport and not also activate the `websocket` feature?